babel.config.js

```diff
@@ -69,7 +69,23 @@ module.exports = function (api) {
           pragma: 'h',
         },
       ],
-      'inline-react-svg',
+      [
```

Comment:
nitpick (non-blockin): Consider importing this config into the babel.config.js as as well as the Storybook configuration.

Reply:
What do you mean?

Reply:
I thought about it, and it's actually not necessary and overkill. All I meant was have the config in one file and import it in both.